tests/test_importer.py

```diff
@@ -105,14 +88,17 @@ def test_sources(data_import, data_folder):
                 'CompositionChild', 'OrganizationName',
                 'MembershipOrganizationMember', 'MembershipOrganizationOrganization',
                 'MembershipPersonOrganization', 'MembershipPersonMember',
-                'CompositionParent',
+                'CompositionParent', 'EmplacementSite', 'EmplacementStartDate',
+                'EmplacementOrganization'
             ])
             permitted_person_set = set(['PersonName'])
             permitted_incident_set = set([
                 'ViolationStartDate', 'ViolationStatus', 'ViolationType',
                 'ViolationFirstAllegation', 'ViolationDescription',
                 'ViolationEndDate', 'ViolationLastUpdate',
-                'ViolationPerpetratorClassification'
+                'ViolationPerpetratorClassification',
+                'ViolationLocationDescription',
+                'ViolationPerpetratorOrganization'
```

Comment:
These were sourced before, so I'm not sure why they weren't in the permitted array initially.